embrace-android-otel-java/src/main/kotlin/io/embrace/android/embracesdk/otel/java/EmbraceOtelJavaExtensions.kt

```diff
@@ -0,0 +1,51 @@
+package io.embrace.android.embracesdk.otel.java
+
+import io.embrace.android.embracesdk.Embrace
+import io.embrace.opentelemetry.kotlin.ExperimentalApi
+import io.embrace.opentelemetry.kotlin.aliases.OtelJavaAttributeKey
+import io.embrace.opentelemetry.kotlin.aliases.OtelJavaLogRecordExporter
+import io.embrace.opentelemetry.kotlin.aliases.OtelJavaOpenTelemetry
+import io.embrace.opentelemetry.kotlin.aliases.OtelJavaSpanExporter
+import io.embrace.opentelemetry.kotlin.logging.export.toOtelKotlinLogRecordExporter
+import io.embrace.opentelemetry.kotlin.toOtelJavaApi
+import io.embrace.opentelemetry.kotlin.tracing.export.toOtelKotlinSpanExporter
+
+/**
+ * Adds an [OtelJavaSpanExporter] that OTel Spans will be exported to after completion.
+ */
+@OptIn(ExperimentalApi::class)
+fun Embrace.addJavaSpanExporter(spanExporter: OtelJavaSpanExporter) {
+    this.addSpanExporter(spanExporter.toOtelKotlinSpanExporter())
+}
+
+/**
+ * Adds an [OtelJavaLogRecordExporter] that OTel LogRecords will be exported to after completion.
+ */
+@OptIn(ExperimentalApi::class)
+fun Embrace.addJavaLogRecordExporter(logRecordExporter: OtelJavaLogRecordExporter) {
+    this.addLogRecordExporter(logRecordExporter.toOtelKotlinLogRecordExporter())
+}
+
+/**
+ * Returns an [OtelJavaOpenTelemetry] that provides working [Tracer] implementations that will record spans that fit into the Embrace data
+ * model.
+ */
+@OptIn(ExperimentalApi::class)
+fun Embrace.getJavaOpenTelemetry(): OtelJavaOpenTelemetry {
+    // TODO: Should we return noop if getOpenTelemetryKotlin is noop?
```

Comment:
This TODO comment should be addressed, ideally by type-checking `this.getOpenTelemetryKotlin` and returning `OtelJavaOpenTelemetry.noop()` if needed, as this avoids running through all the adapters

Reply:
We can't typecheck against `NoopOpenTelemetry`, it's an internal object. Should we expose it in otel-kotlin?